src/content/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/alerting-new-relic-mobile-apps.mdx

```diff
@@ -2,7 +2,7 @@
 title: Alerting with New Relic mobile apps
 tags:
   - Mobile apps
-  - New Relic Mobile apps
+  - New Relic m apps
```

Comment:
pretty sure this should read "mobile apps"

Reply:
Yikes, good catch